web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -386,6 +390,77 @@ describe("profile - shared", () => {
     });
   });
 
+  describe("Special Note Alert for Businesses Formed outside the Navigator", () => {
+    it.each(["FOREIGN", "STARTING"])(
+      "shows Note alert for FOREIGN and STARTING businesses that set DateOfFormation but did NOT pay",
+      (persona) => {
+        const business = generateBusinessForProfile({
+          formationData: generateFormationData({
+            completedFilingPayment: false,
+          }),
+          profileData: generateProfileData({
+            dateOfFormation: "2020-8-8",
+            businessPersona: persona as BusinessPersona,
+          }),
+        });
+        renderPage({ business });
+        expect(
+          screen.getByText(Config.profileDefaults.default.noteForBusinessesFormedOutsideNavigator)
+        ).toBeInTheDocument();
+      }
+    );
+
+    it("shows the Note Alert for OWNING businesses always", () => {
+      const business = generateBusinessForProfile({
+        formationData: generateFormationData({
+          completedFilingPayment: false,
```

Comment:
Dust - to fully bulletproof the `isFormedOutsideNavigator` function, should we update this to true? Wondering this because, let's say in the future we remove `business.profileData.dateOfFormation)` from the condition, the function would be failing because the first half of the conditional is true. 

  if (
    (business.formationData.completedFilingPayment === false && business.profileData.dateOfFormation) ||
    business.profileData.businessPersona === "OWNING"
  ) {

Reply:
Sure, that feels a bit more robust.